woocommerce/woocommerce-android
# Shipping Labels: Customs screen UI part 1

Part 2 of #3526, this adds the UI implementation for the customs screen, this doesn't implement any interaction with the UI, nor loading real data, those will be handled in a separate PR, as this one is already big.

|  |  |
|----|-----|
|![Screenshot_1621617535](https://user-images.githubusercontent.com/1657201/119174908-12817480-ba61-11eb-8d64-84ccc9feb42c.png)|![Screenshot_1621617651](https://user-images.githubusercontent.com/1657201/119175058-49f02100-ba61-11eb-91b0-811f8ac36bbe.png)|

#### Testing
1. Open order details of an order that's eligible for shipping label creation.
2. Click on Create shipping label.
3. Notice that the customs step is visible on debug builds.
4. Pass the first step.
5. Click on Continue in the customs step.
6. Check the UI (all data is fake for now).
7. Confirm that there is no regression in the other parts of the shipping labels form is enough (for this step, please note that the order has to be destined to US for the purchase to work)

Update release notes:
